docs/manuals/administrator-manual.md

```diff
@@ -0,0 +1,82 @@
+# Beheerdershandleiding ZGW Office Add-In
+| versie | datum      | wijziging                                                      |
+|--------|------------|----------------------------------------------------------------|
+| 0.1    | 2025-07-07 | Opzet van deze beheerdershandleiding in de context van MS Word |
+
+## Inleiding
+Deze beheerdershandleiding is bedoeld voor systeembeheerders van gemeenten die de ZGW Office Add-In beheren. De 
+add-in is een hulpmiddel dat de integratie van Microsoft Office applicaties met een zaaksysteem gebaseerd op de ZGW 
+APIs mogelijk maakt.
+
+## Uitgangssituatie
+De uitgangssituatie waar deze handleiding op is gebaseerd, is dat de systeembeheerder werkt met Microsoft 365 Office 
+voor een gemeente die de ZGW Office Add-In heeft geïnstalleerd op een omgeving die toegankelijk is voor de medewerkers 
+van de gemeente.
```

Comment:
```suggestion
voor een gemeente die de ZGW Office Add-In heeft geïnstalleerd..
```

I don't think the rest of the sentence adds any value? Why would medewerkers not have access to this environment? What does that even matter really in this context?